docs/dictionary/function/revBrowserOpenCef.lcdoc

```diff
@@ -9,7 +9,9 @@ Opens and initialises a new Chrome-based browser.
 
 Introduced: 6.7
 
-OS: mac, windows
+Deprecated: In LiveCode 8.0+, this function is no longer supported on Mac.
```

Comment:
@livecodepanos This will have to go in a `Changes:` section below the description for the moment, as adding a `Deprecated:` element will cause it to appear with the deprecated icon, implying it is deprecated on all platforms